src/firefly/test/edu/caltech/ipac/firefly/server/query/PhaseFoldedLightCurveTest.java

```diff
@@ -0,0 +1,56 @@
+package firefly.test.edu.caltech.ipac.firefly.server.query;
+
+import edu.caltech.ipac.astro.IpacTableException;
+import edu.caltech.ipac.firefly.server.query.PhaseFoldedLightCurve;
+import edu.caltech.ipac.firefly.server.util.ipactable.DataGroupReader;
+import edu.caltech.ipac.util.DataGroup;
+import edu.caltech.ipac.util.DataObject;
+import edu.caltech.ipac.util.DataType;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by ymei on 8/25/16.
+ */
+public class PhaseFoldedLightCurveTest {
+
+    private static final String TEST_ROOT = "test"+ File.separatorChar;
+    private static String inFileName = "/hydra/cm/firefly/src/firefly/test/edu/caltech/ipac/firefly/server/query/AllWISE-MEP-m82-2targets-10arsecs-oneTarget.tbl";
+    //private static String inFileFullName =  TEST_ROOT + inFileName;
```

Comment:
Please, make the test to read the file relative to the test folder. This file path is absolute and note necessary match